packages/front-end/components/CustomFields/CustomFieldInput.tsx

```diff
@@ -49,15 +55,16 @@ const CustomFieldInput: FC<{
             }
           }
         });
-        form.setValue("customFields", currentCustomFields);
+        //updateAllCustomFields(currentCustomFields);
+        setCustomFields(currentCustomFields);
         setLoadedDefaults(true);
       }
     }
-  }, [availableFields, form, loadedDefaults, currentCustomFields]);
+  }, [availableFields, loadedDefaults, currentCustomFields, setCustomFields]);
 
   const updateCustomField = (name, value) => {
     currentCustomFields[name] = value;
-    form.setValue("customFields", currentCustomFields);
+    setCustomFields(currentCustomFields);
```

Comment:
Minor: Should not mutate` currentCustomFields`.  Instead, use the spread operator.